pkg/isb/message.go

```diff
@@ -73,6 +74,23 @@ type Header struct {
 	Headers map[string]string
 }
 
+// MessageID is the message ID of the message which is used for exactly-once-semantics.
+type MessageID struct {
```

Comment:
Will this change impact existing pipelines?

Reply:
No